hack/util.sh

```diff
@@ -74,6 +74,7 @@ function configure_os_server {
 	openshift start \
 	--write-config=${SERVER_CONFIG_DIR} \
 	--create-certs=false \
+	--dns="tcp://${API_HOST}:53" \
```

Comment:
> Ok. Can you prune the start_os* calls to remove flags that aren't honored when specifying config? I would have expected validation to fail on the CLI when you pass a flag that's going to be ignored ....

Separate bit of work I think.  This breaks the only "fast" e2e path we have for local testing.